.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: node_js
 cache: yarn
 script:
-  - yarn --version
-  - yarn check --integrity
-  - yarn test
+- yarn --version
+- yarn check --integrity
+- yarn test
 before_install:
-  - yarn global add greenkeeper-lockfile@1
+- yarn global add greenkeeper-lockfile@1
```

Comment:
don't remove correct spacing in yaml file

Reply:
hmm, this wasn't touched by hand only changed by the `travis` command line tool.
I would side with travis as this is a file for their CI

Reply:
oh. normally i would trust travis cli but will not accept their own "spacing". it is under developed

Reply:
problem is if you use their tool to change anything in the file it will just change it back, why fight?

Reply:
I just tested by manually adding spaces back and then running:
```
travis encrypt  --add
```
and the spaces were removed

Reply:
no fight. your explanation was enough

Reply:
I didn't mean me fight you, I meant why fight the tool.
Sorry if any misunderstanding 👍

Reply:
i understood it correctly ;)